content/changelog/2024-02-08-sozu-0.15.19.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sōzu 0.15.19 is available, 1.0 is coming
+date: 2024-02-08
+tags:
+  - sozu
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+  - name: Florentin Dubois
+    link: https://github.com/FlorentinDUBOIS
+    image: https://github.com/FlorentinDUBOIS.png?size=40
+description: Better, faster, stronger
```

Comment:
are you sure ?

Reply:
Was a mistake from previous entry, fixed